setup.py

```diff
@@ -59,6 +59,14 @@
 ISRELEASED          = False
 VERSION             = '%d.%d.%d' % (MAJOR, MINOR, MICRO)
 
+# keep this consistent with the `Programming Language :: Python :: ...` classifiers above
+if sys.version_info >= (3, 10):
```

Comment:
By changing this to 3.10, you've already made this inconsistent!

Reply:
Ha, good point! Updated, I can rebase and squash these commit when we're done.